src/DroneSafetyApi/Models/IDataPointRepository.cs

```diff
@@ -6,10 +6,10 @@
 
 namespace DroneSafetyApi.Models
 {
-    public interface IHazardsRepository
+    public interface IDataPointRepository
```

Comment:
Disagree, more natural to use the singular. Eg a box for books is a "book box" not a "books box".